### PR TITLE
Update Docker CI

### PR DESCRIPTION
**Pull request by mbrookhart**
Adding a lot of updates to CI docker images, including onnx, arm, tensorflow, tflite, etc. Fixes a couple of bugs with the update.

### CODE REVIEW

File: docker/install/ubuntu_install_onnx.sh [hunk 1]
```diff
@@ -20,9 +20,10 @@ set -e
 set -u
 set -o pipefail
 
-# fix to certain version for now
-pip3 install onnx==1.6.0
-pip3 install onnxruntime==1.0.0
+# We need to fix the onnx version because changing versions tends to break tests
+# TODO(mbrookhart): periodically update
+pip3 install onnx==1.8.0
```
Comment (areusch) [inline docker/install/ubuntu_install_onnx.sh:25]: can you also update `python/gen_requirements.py`?

Comment (mbrookhart) [reply docker/install/ubuntu_install_onnx.sh:25]: I'm not sure what there is to update? gen_requirements.py doesn't constrain the version of ONNX, and the goal here is to maintain backwards compatibility with old versions of onnx, that's why the importer supports opsets.

Comment (areusch) [reply docker/install/ubuntu_install_onnx.sh:25]: ah ignore me, we were constraining this separately

**Comment by areusch** [comment]
```
+ echo 'INFO: NODE_NAME=sampl.aladdin.cuda1 EXECUTOR_NUMBER=0'
INFO: NODE_NAME=sampl.aladdin.cuda1 EXECUTOR_NUMBER=0`
```

looks like another SAMPL node @tqchen. do we want to keep this in for diversity of testing or remove it? my vote is to remove it since it is a probabilistic failure scenario rather than a deterministic one. i think our CI is so long that we can't afford to allow code to merge that isn't expected to deterministically pass a following run.

**Comment by mehrdadh** [comment]
@mbrookhart I just tested it for both `host` and `mps2` target and it passes. It could be a flaky timeout.
Please re-trigger the test.

File: Jenkinsfile [hunk 2]
```diff
@@ -45,12 +45,12 @@
 
 // NOTE: these lines are scanned by docker/dev_common.sh. Please update the regex as needed. -->
 ci_lint = "tlcpack/ci-lint:v0.66"
-ci_gpu = "tlcpack/ci-gpu:v0.75"
-ci_cpu = "tlcpack/ci-cpu:v0.74"
+ci_gpu = "mbrookhart/ci-gpu:v0.2"
```
Comment (areusch) [inline Jenkinsfile:48]:
```suggestion
ci_gpu = "tlcpack/ci-gpu:v0.76"
```

File: Jenkinsfile [hunk 3]
```diff
@@ -45,12 +45,12 @@
...
+ci_cpu = "mbrookhart/ci-cpu:v0.2"
```
Comment (areusch) [inline Jenkinsfile:49]:
```suggestion
ci_cpu = "tlcpack/ci-cpu:v0.75"
```

File: Jenkinsfile [hunk 4]
```diff
@@ -45,12 +45,12 @@
...
 ci_wasm = "tlcpack/ci-wasm:v0.71"
 ci_i386 = "tlcpack/ci-i386:v0.73"
-ci_qemu = "tlcpack/ci-qemu:v0.05"
-ci_arm = "tlcpack/ci-arm:v0.05"
+ci_qemu = "mbrookhart/ci-qemu:v0.2"
```
Comment (areusch) [inline Jenkinsfile:52]:
```suggestion
ci_qemu = "tlcpack/ci-qemu:v0.06"
```

File: Jenkinsfile [hunk 5]
```diff
@@ -45,12 +45,12 @@
...
+ci_arm = "areusch1/ci-arm-staging:v0.05"
```
Comment (areusch) [inline Jenkinsfile:53]:
```suggestion
ci_arm = "tlcpack/ci-arm:v0.06"
```

File: tests/python/frontend/onnx/test_forward.py [hunk 6]
```diff
@@ -2686,62 +2691,58 @@ def repeat(N, D):
...
-            (1, 1) + repeat(5, D),
-            (1, 1) + repeat(3, D),
-            (1, 1) + repeat(9, D),
-            None,
-            repeat(3, D),
-            repeat(2, D),
-            repeat(1, D),
-            auto_pad="SAME_UPPER",
-        )
+        ## TODO(mbrookhart): renable autopad tests when CI ONNX
```
Comment (areusch) [inline tests/python/frontend/onnx/test_forward.py:2709]: per discussion, this will be a follow-on PR

**Review by areusch — APPROVED** [review]
thanks @mbrookhart ! please update the tracking issue when you merge

**Comment by mbrookhart** [comment]
Thanks @areusch @mehrdadh

**Pull request merged**